sinonjs/sinon
# Add unimported to workflow

This PR adds a check for unimported files to the GitHub Actions workflow, using https://github.com/smeijer/unimported

As always, the PR is easier to understand when you read the commits individually.

#### How to verify

1. Add a file to `lib/` that is not used
2. `npm run unimported`
3. Observe that there's an error
4. Remove the file
5. `npm run unimported`
6. Observe that there is no error
